.github/workflows/check-typescript-projects.yml

```diff
@@ -69,3 +71,113 @@ jobs:
         run: npm install ${{steps.download.outputs.download-path}}/bpmn-visualization*.tgz
       - name: Build project
         run: npm run build
+      - name: Upload project archive
+        if: github.event_name == 'pull_request'
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{matrix.project}}-${{matrix.bv-npm-package}}
+          path: ${{github.workspace}}/projects/${{matrix.project}}/dist
+
+  # WARN: use a short job-id to ensure the surge.sh subdomain is valid i.e. up to 63 characters (the surge-preview action includes the job-id in the generated surge domain)
```

Comment:
ℹ️ we may remove this comment, everything is explain in the `bonitasoft/actions` issue